<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowCamera</class>
 <widget class="QMainWindow" name="MainWindowCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>955</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simox - VirtualRobot - Show Robot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>400</width>
         <height>250</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonLoad">
          <property name="text">
           <string>Select Robot File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Joints</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBoxJoint"/>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBoxRobotNodeSet"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Select RobotNodeSet</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Select RobotNode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QSlider" name="horizontalSliderPos">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="pageStep">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelMinPos">
               <property name="text">
                <string>-1.57</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumberJointValue">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="smallDecimalPoint">
                <bool>true</bool>
               </property>
               <property name="digitCount">
                <number>5</number>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Filled</enum>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelMaxPos">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>1.57</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Joint Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Robot y-position</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSliderRobotY">
          <property name="minimum">
           <number>-750</number>
          </property>
          <property name="maximum">
           <number>750</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>260</y>
         <width>400</width>
         <height>621</height>
        </rect>
       </property>
       <widget class="QCheckBox" name="checkBoxDepthCam1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>391</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Depth image (linear scale, clipping at                              )</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="cam1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>400</width>
          <height>260</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>11</width>
          <height>0</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxDepthCam2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>391</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Depth image (nonlinear (                       *z-buffer))</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="cam2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>360</y>
          <width>400</width>
          <height>260</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxNonLinFactor">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>330</y>
          <width>69</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxDepthLinClip">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>91</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimum">
         <double>11.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>4000.000000000000000</double>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxShowDepthVoxel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Show depth voxel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
